lib/webdriverio.js

```diff
@@ -230,6 +234,14 @@ let WebdriverIO = function (args, modifier) {
             })
     }
 
+    function createRequestHanlderDecorator (opts) {
```

Comment:
I am concerned about this solution because once you take a screenshot the requestHandler is decorated and will used for the next requests even though they aren't screenshot requests